docs/LogGuide.md

```diff
@@ -1,3 +1,6 @@
+<!-- Disable MD004, markdown-magic generates TOC with - instead of * -->
```

Comment:
MD004's default is "consistent", so you could not disable that by making the other bullet points use `-` instead of `*`.

Reply:
Thanks. I'm not very familiar with markdownlint rules.